src/store/utilities/importModel.js

```diff
@@ -1,95 +1,96 @@
-import idFactory from './generateId'
+import idFactory from './generateId';
 
-const d3 = require('d3');
+export default function importModel(context, payload) {
+  // intializr a versionNumber if the app is running in embedded mode
+  if (window.api) { window.versionNumber = 0; }
 
-export default function importModel (context, payload)  {
-    if (window.api) { window.versionNumber = 0; }
-    // GEOMETRY
-    const geometry = JSON.parse(JSON.stringify(payload.data.stories.map(s => s.geometry)));
-    for (var i = 0; i < geometry.length; i++) {
-        geometry[i].faces = geometry[i].faces.map((f) => {
-            return {
-                id: f.id,
-                edgeRefs: f.edge_ids.map((edge_id, i) => {
-                    return {
-                        edge_id: edge_id,
-                        reverse: !f.edge_order[i]
-                    }
-                })
-            }
-        });
-
-        geometry[i].edges = geometry[i].edges.map((e) => {
-            return {
-                id: e.id,
-                v1: e.vertex_ids[0],
-                v2: e.vertex_ids[1]
-            };
-        });
+  // GEOMETRY
+  const geometry = JSON.parse(JSON.stringify(payload.data.stories.map(s => s.geometry)));
+  for (var i = 0; i < geometry.length; i++) {
+    geometry[i].faces = geometry[i].faces.map((f) => {
+      return {
+        id: f.id,
+        edgeRefs: f.edge_ids.map((edge_id, i) => {
+          return {
+            edge_id: edge_id,
+            reverse: !f.edge_order[i]
+          }
+        })
+      }
+    });
 
-        geometry[i].vertices = geometry[i].vertices.map((v) => {
-            return {
-                id: v.id,
-                x: v.x,
-                y: v.y,
-                X: v.x,
-                Y: v.y
-            };
-        });
-    }
+    geometry[i].edges = geometry[i].edges.map((e) => {
+      return {
+        id: e.id,
+        v1: e.vertex_ids[0],
+        v2: e.vertex_ids[1]
+      };
+    });
 
-    // MODELS
-    payload.data.stories.forEach((story) => {
-        story.geometry_id = story.geometry.id;
-        delete story.geometry;
+    geometry[i].vertices = geometry[i].vertices.map((v) => {
+      return {
+        id: v.id,
+        x: v.x,
+        y: v.y,
+        X: v.x,
+        Y: v.y
```

Comment:
This used to be required for clipper, but can actually be removed now. Good catch!